examples/main-xml/pom.xml

```diff
@@ -113,5 +113,21 @@
             </plugin>
         </plugins>
     </build>
-
+    <profiles>
+        <profile>
+            <id>camel.debug</id>
```

Comment:
I'm wondering if it is really needed to prefix it by camel? it sounds like a pleonasm to me

Reply:
I think it helps to make it clear that the profile is related to Camel. It avoids potential collision for the user adding profiles for other kind of debug, like a java remote debug, or a springboot specific debug.